sdk/core/azure-core/azure/core/exceptions.py

```diff
@@ -107,6 +107,27 @@ class ODataV4Format(object):
 
     http://docs.oasis-open.org/odata/odata-json-format/v4.0/os/odata-json-format-v4.0-os.html#_Toc372793091
 
+    Example of JSON:
+    error": {
+            "code": "ValidationError",
+            "message": "One or more fields contain incorrect values: ",
+            "details": [
+                {
+                    "code": "ValidationError",
+                    "target": "representation",
+                    "message": "Parsing error(s): String '' does not match regex pattern '^[^{}/ :]+(?: :\\\\d+)?$'.
+                    Path 'host', line 1, position 297."
+                },
+                {
+                    "code": "ValidationError",
+                    "target": "representation",
+                    "message": "Parsing error(s): The input OpenAPI file is not valid for the OpenAPI specificate
+                    https: //github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md
+                    (schema https://github.com/OAI/OpenAPI-Specification/blob/master/schemas/v2.0/schema.json)."
+                }
+            ]
+        }
```

Comment:
Would be worth checking how this renders in sphinx (if it's actually in the public docs).
It may need additional `code-block` tags for the formatting to work.

Reply:
From my experiment, .. code-block:: tag will make the json invisible in sphinx